src/hotspot/share/classfile/verifier.cpp

```diff
@@ -3149,7 +3149,7 @@ void ClassVerifier::verify_return_value(
   if (return_type == VerificationType::bogus_type()) {
```

Comment:
return_type is the type expected from the function, which here is void (I guess void == bogus_type, but ok).
So the message is correct.